content/en/docs/languages/cpp/quickstart.md

```diff
@@ -114,7 +114,7 @@ $ cmake -DgRPC_INSTALL=ON \
       -DgRPC_BUILD_TESTS=OFF \
       -DCMAKE_INSTALL_PREFIX=$MY_INSTALL_DIR \
       ../..
-$ make -j
+$ make -j 8
```

Comment:
I'm fine with setting to `make -j4` -  that should work for most people and still gives a good speedup if you have more cores available.
I think users can figure it out and adjust the parameter quite easily if needed.

Note that line 152 also has `make -j` - let's update that to -j4 for consistency as well.

Reply:
Done. Everywhere there was a `-j` I changed it to `-j 4`. PTAL